catalog/files/impl/src/main/java/org/projectnessie/catalog/files/adls/AdlsObjectIO.java

```diff
@@ -32,43 +32,28 @@
 import java.util.stream.Collectors;
 import org.projectnessie.catalog.files.api.ObjectIO;
 import org.projectnessie.storage.uri.StorageUri;
-import org.slf4j.Logger;
-import org.slf4j.LoggerFactory;
 
 public class AdlsObjectIO implements ObjectIO {
-  private static final Logger LOGGER = LoggerFactory.getLogger(AdlsObjectIO.class);
-
   private final AdlsClientSupplier clientSupplier;
 
   public AdlsObjectIO(AdlsClientSupplier clientSupplier) {
     this.clientSupplier = clientSupplier;
   }
 
   @Override
-  public void ping(StorageUri uri) throws IOException {
+  public void ping(StorageUri uri) {
     AdlsLocation location = adlsLocation(uri);
 
     DataLakeFileSystemClient fileSystem = clientSupplier.fileSystemClient(location);
-    try {
-      fileSystem.getAccessPolicy();
-    } catch (Exception e) {
-      throw new IOException(e);
-    }
+    fileSystem.getAccessPolicy();
```

Comment:
Can you sneak in a change to `getProperties()` - see [here](https://github.com/projectnessie/nessie/pull/9086#discussion_r1679410901)

Reply:
#9134